CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### Apr 23, 2025
+
+-  Add ability to select multiple shapefiles. [DESENG-769](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-769)
```

Comment:
Just a slight perspective thing, but you added the changes for the admin section. The public would be something like 'Added ability to display multiple shapefile colours per project on the project map.'

Reply:
Yes, you're absolutely right. I'll update